hadoop-ozone/dist/src/main/smoketest/ec/basic.robot

```diff
@@ -38,7 +38,7 @@ Prepare For Tests
 Test Bucket Creation
     ${result} =     Execute             ozone sh volume create /${prefix}vol1
                     Should not contain  ${result}       Failed
```

Comment:
Why do we need to pass ratis options here?

Reply:
Bydefault, server should use ratis right

Reply:
Correct. I have changed bucket default replication config to `null` instead of `RATIS`.
If bucket-level replication config is missing, cluster-level config will be used.
If bucket-level replication config is `null`, it will not show in the result of listing buckets.

Reply:
> Why do we need to pass ratis options here?

Because if no bucket-level replication config is set, list bucket will not show replicationConfig anymore. Since we are checking the result of list bucket, we need to pass ratis option here.

Reply:
> Bydefault, server should use ratis right

I have added `Test Key Creation Default Bucket` to verify this.

Reply:
@kaijchen, Thanks for proving the details. I am not sure I fully get it here, but here is my another question to get it clarified:
What if you don't pass ratis option here in the below line?
${result} =     Execute             ozone sh bucket create --replication 3 --type RATIS /${prefix}vol1/${prefix}ratis

Does this still create ratis key? I understand you already added a test with default. If we remove the options above, this should also be like default. But you are just testing these options as well. Is this correct?  ( I am not very good in understanding these script based tests yet. )

I am trying to make sure the behavior is same as expected.

Reply:
Thanks for the details. Yes, this is expected behavior. I am fine with this. Compat issue you are pointing is within the ec branch, so no issues with that.